### ISSUE TITLE
Tags

### ISSUE THREAD

**Issue by benguhin** [cfpb/design-manual#344]
This came up a while ago in discussions about button styles, as "tags" are often actionable, but we may need an additional style for tags that aren't actionable. Assigning to @ohsk and the other folks on Complaint Analytics

**Comment by duelj** [comment]
Just wanted to share V1's current uses for tags which we currently have labeled 'topics.' 
- The interaction of these tags is that when clicked users are lead to a pages of post filtered by that tag. 
- There was a question over the accessibility of the gold being a problem. Wondered if @JenniferHoran had any input on the bullet and underline being a non-accessible color in this approach. Is it a problem?

| Page Type | Visual Example |
| --- | --- |
| Blog | ![screen shot 2015-10-05 at 5 10 00 pm](https://cloud.githubusercontent.com/assets/6873734/10293695/ff835f6a-6b83-11e5-8c02-05b375a8dade.png) |
| Event | ![screen shot 2015-10-05 at 5 10 07 pm](https://cloud.githubusercontent.com/assets/6873734/10293703/0859720a-6b84-11e5-9416-2ec4651ff561.png) |

**Comment by ohsk** [comment]
Some other examples of tags / tag-like objects.

#### Tumblr / #hashtag format

![image](https://cloud.githubusercontent.com/assets/1689222/10315817/7ee61a08-6c2a-11e5-8a6b-ddf903cb162e.png)

#### Dribbble / pill-like format

![image](https://cloud.githubusercontent.com/assets/1689222/10315845/9aa849a0-6c2a-11e5-9914-670c5d66cd43.png)

#### Pinterest / interactive pill-like format

![image](https://cloud.githubusercontent.com/assets/1689222/10315874/c949147e-6c2a-11e5-89ce-7d6f1a041226.png)

#### NPR / trending topics, block-like format

![image](https://cloud.githubusercontent.com/assets/1689222/10315947/261c545e-6c2b-11e5-8191-e72eb5cb7581.png)

**Comment by benguhin** [comment]
:heart: the term "tag-like object"

The gray outline on the tags in the the dribble example is helpful because it allows them to be used over a gray background

**Comment by ohsk** [comment]
Overkill trying to use iconography? We've been discussing the need to differentiate filters vs. tags in [our use case](https://github.cfpb.gov/ConsumerResponse/complaint-analytics-search-tool/issues/256#issuecomment-55910)...

![image](https://cloud.githubusercontent.com/assets/1689222/10318356/fdb8cd24-6c36-11e5-909c-c8067181388d.png)

**Comment by ielerol** [comment]
I finally have a reasonable draft of [guidelines for tagging usage.](https://docs.google.com/a/cfpb.gov/document/d/1RmSGREaL6g-rkb5c-YFEiF_V6KjacpRIgbPgTFSTZcY/edit?usp=sharing)

This is another issue where I think the context and use cases for the tags can have a lot of impact on how they need to be used, so if anyone has used tags in a situation that doesn't seem to be covered by this document, please chime in.

**Comment by schaferjh** [comment]
Looks good @ielerol -- I made a few comments but generally I think you've got the bases covered. :+1:

**Comment by keelerr** [comment]
I also made a few comments, but agree it's looking good. :+1:

**Comment by duelj** [comment]
I know there is already some good exploration on this but, I just was perusing Google's Material Design and found that they have a sort of multipurpose tag that they refer to as "chips." The style is nice, as the full radius sides really set it apart from buttons and work nicely with the circle icons. Thought it could be good reference for us as we move forward. 

<img width="758" alt="screen shot 2015-10-26 at 4 17 38 pm" src="https://cloud.githubusercontent.com/assets/6873734/10741267/3f4295c4-7bfd-11e5-8085-36f2edd48b8b.png">

**Comment by ohsk** [comment]
I could get behind those! I think a bit more distinction from the buttons is worthwhile. Here's what the full radius looks like applied to our filter pills in the current CR Search demo site:

![image](https://cloud.githubusercontent.com/assets/1689222/10741615/21a31b9a-7bff-11e5-9d57-8589fc60d9b0.png)

**Comment by stephanieosan** [comment]
Just wanted to share an experience from testing. 

We tested a mobile prototype (with a handful of people) that included the gold-style tags. 
<img width="377" alt="screen shot 2016-01-28 at 12 39 51 pm" src="https://cloud.githubusercontent.com/assets/6433178/12653180/4bb75cb4-c5bc-11e5-9d88-b00ab506ed3f.png">

We found that users either didn't notice them at all, or if they did, they didn't recognize them as clickable. We think we should pursue a more clickable-looking tag style for the design manual standards.

**Comment by adinneen** [comment]
@amycesal Based on our discussion sounds like we are talking about three things in this thread: 

**1. A pill based filtering style which is completely separate from "tagging."**  
_Example:_ In the Complaint Analytics Search Tool - We style filters like pills to how they combine with a keyword search.  Pill filters are applied by checking a box on the left hand filter bar.  
<img width="647" alt="screen shot 2016-02-04 at 1 18 20 pm" src="https://cloud.githubusercontent.com/assets/12372787/12824982/e9b6acdc-cb41-11e5-9fa5-50728635e178.png">

**2. A very "clickable" style tag**
_Example:_ Ask CFPB uses a clickable feeling tag style to show what topic areas complaints are associated with and then let users pull back items on the same topic.  
<img width="668" alt="screen shot 2016-02-04 at 1 22 31 pm" src="https://cloud.githubusercontent.com/assets/12372787/12825076/79ec195e-cb42-11e5-8d3f-bbfde2bc5343.png">

**3. A "non-clickable" style tag**
_Example:_ The V1 re-design team is using a "tag" style to show topics that does not feel quite as clickable.  Right now I believe these are clickable but they could be non-clickable. 
![image](https://cloud.githubusercontent.com/assets/12372787/12825135/c5e8ca78-cb42-11e5-8279-fd747b149b0d.png)
Another option the Complaint Analytics team is exploring is a # type style that would be non-clickable
<img width="456" alt="screen shot 2016-02-04 at 1 26 46 pm" src="https://cloud.githubusercontent.com/assets/12372787/12825183/faba27f6-cb42-11e5-8aac-71756779e811.png">

**Comment by caheberer** [comment]
Tags are live on AskCFPB and as part of filtering in the News section, as well as internal tools. 

It doesn't look like this is standardized. Are there any CF components built? @Scotchester @jimmynotjim

**Comment by jimmynotjim** [comment]
@caheberer No, it's never made it to Capital Framework. Once we have a standard we can add it to our backlog and start to get it created.
